content/en/docs/tasks/traffic-management/istioctl-describe/index.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Understanding your mesh with with istioctl describe [Experimental]
+description: Shows you how to use istioctl to determine the traffic management applied to a pod.
+weight: 90
+keywords: [traffic-management, istioctl, debugging]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+In Istio 1.3, we included the experimental `describe pod` sub-command for `istioctl`. We designed this tool to help find and
+understand the configuration that impacts the traffic to a pod.  This task shows you how
+to use the experimental sub-command.
+
+## `istioctl experimental describe pod` tutorial
+
+First, let us [deploy Bookinfo](/docs/examples/bookinfo/).
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+$ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+$ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].port}')
+$ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+$ echo Bookinfo at $GATEWAY_URL/productpage
```

Comment:
what does this echo do?

Reply:
The bookinfo instructions at https://istio.io/docs/examples/bookinfo/#confirm-the-app-is-accessible-from-outside-the-cluster say "You can also point your browser to http://$GATEWAY_URL/productpage to view the Bookinfo web page."  The echo prints this value, perhaps I should re-word the message.